integrations/dd-sdk-android-okhttp/src/main/kotlin/com/datadog/android/okhttp/internal/OkHttpHttpSpecRequestInfo.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+package com.datadog.android.okhttp.internal
+
+import com.datadog.android.api.instrumentation.network.ExtendedRequestInfo
+import com.datadog.android.api.instrumentation.network.HttpSpecRequestInfo
+import com.datadog.android.rum.internal.net.buildResourceId
+import okhttp3.Request
+
+@Deprecated("This code will be replaced with RequestExt.kt and RequestInfo in the further releases.")
```

Comment:
since it is `internal` anyway, do we need to have `@Deprecated`? or it is just for a memo at the call sites?